### PR TITLE
Feature: Setup components

### PR DESCRIPTION
**Pull request by Akorede3133**
In the PR, the following were affected;
* Add 2 routes in your application, each should render a page component:

- [ ] / 
- [ ]  /categories

* Created components inside of the /components directory:
  

- [ ] Contains individual book state, as well as a button to delete the book
- [ ]   Contains a list to renders individual books
- [ ]   Contains a form to create new books
- [ ]   Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,174 @@
...
+    <li><a href="https://www.css3.com/">CSS3</a></li>
+  </ul>
+</details>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<details>
+<summary>Javascript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+  </ul>
```
Comment (joel-kalema) [inline README.md]: - [ ] Good job initializing the project with components 🥇, however, let's remove the `HTML` `CSS3` and `javascript` as we are using only React to make your README more professional 👍 

Screenshot 2023-08-17 174012

File: README.md [hunk 2]
```diff
@@ -1,70 +1,174 @@
...
+I would like to thank Microverse for this great opportunity!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Akorede3133/bookstore/blob/dev/LICENSE) licensed.
+
```
Comment (joel-kalema) [inline README.md]: - [ ] Also, make sure you fix the link to your LICENSE to make your README more professional 👍 

Screenshot 2023-08-17 171648

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Akorede3133 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

0bd223a2b156e0eaf61a713e84c3d3b7



## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @Akorede3133 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good work making the changes requested 👍 


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**